test/items_formatter_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ItemsFormatterTest do
+  use ExUnit.Case
+  doctest ToDo
+
+  test "to do menu options displayed with no items" do
+    assert ToDo.ItemsFormatter.items_for_display([ 'Go to yoga tonight', 'make to do list app']) ==
```

Comment:
I don't know what you mean by the > but I guess you're saying I could do an Import ToDo.ItemsFormatter maybe?   I've done this in the later PR..